src/ShortcutGuide.tsx

```diff
@@ -22,282 +27,63 @@ const ShortcutGuide: React.FC = () => {
 
 	return (
 		<>
-			<div className="a11y_modal" data-testid="a11y_modal">
+			<div className="shortcut-guide__background" data-testid="shortcut-guide">
 				<div
-					className="a11y_modal_content"
+					className="shortcut-guide__container"
+					tabIndex={0}
 					aria-label="AutoVizually shortcut guide. AutoVizually lets you navigate between
```

Comment:
I'm not so sure that we should have this as a label. Because every time the user places the cursor on this element, the screen reader will read this out loud. Instead, what we could is to use the `aria-describedby` attribute that points to an element on the DOM (visually hidden) that has this info. That way, the screen reads this after a very brief pause, but not every time.

Let me know what you think.